addons/storyshots/src/require_context.js

```diff
@@ -40,6 +41,14 @@ function isRelativeRequest(request) {
   );
 }
 
+function getFullPath(dirname, request) {
+  if (isRelativeRequest(request) || !process.env.NODE_PATH) {
```

Comment:
I don't think it needs to be this complex; `process.env.NODE_PATH` will automatically apply if you require something non-relative, won't it?

So it can just be:

```
return isRelativeRequest(request) ? path.resolve(dirname, request) : dirname;
```

Reply:
Checked it now, and it's not applied. It's just resolved according to the cwd. 🤷‍♂️

Reply:
Oooh, because we need to a `fs.readdirSync` in the directory. Sorry I am bit slow 😊 

Do we not need to worry about other require rules in this case? For instance if I do `require.context('foo')` and I have `node_modules/foo` or `../node_modules/foo` or .. etc

I guess this is better than nothing though!